test/test-network/systemd-networkd-tests.py

```diff
@@ -523,7 +523,7 @@ def wait_address(self, link, address_regex, scope='global', ipv='', timeout_sec=
         else:
             self.assertRegex(output, address_regex)
 
-class NetworkctlTests(unittest.TestCase, Utilities):
+class NetworkctlTests(Utilities):
```

Comment:
I don't see the point. The code was actually more semantically correct before. It of course applies to all similar changes below.